src/store/model-store.js

```diff
@@ -75,4 +85,8 @@ store.registerHandler('NEW_MODEL', data => {
     models.push(model);
 });
 
+store.registerHandler('DELETE_MODEL', data => {
+    store.deleteModelByAddress(data.modelAddress);
```

Comment:
No other stores in this entire project have a store.emitChange(). Are you sure that this needs to be added? @mattmahn @thofmann